packages/components/src/components/telekom/app-header/app-header.tsx

```diff
@@ -218,11 +218,7 @@ export class Header {
       readData(this.mainNavigation),
       this.activeRouteId
     );
-    const isActive = (item) =>
-      rootNode &&
-      rootNode.id === item.id &&
-      !this.visibleMegaMenu &&
```

Comment:
are you sure this was dead code? you can never know. can we keep it, just to be sure? the header has so many use-cases.

Reply:
@maomaoZH i think this was part of your bugfix, what do you think?

Reply:
So the designer wants to keep the selected nav status even when user navigates to other tab. If we keep the code, every time when user navigates to other tab which has mega-menu, the selected status will lose.

Reply:
give me one sec please. dont merge yet

Reply:
https://user-images.githubusercontent.com/23239723/160800784-c36f829e-5d43-4627-b7e7-901d73e968cc.mov

have you noticed the menu is jumping.

Reply:
I pushed a fix for this, thanks for checking!